cloudinit/distros/debian.py

```diff
@@ -311,8 +421,57 @@ def regenerate_locale(locale, sys_path, keyname="LANG", install_function=None):
         LOG.debug("%s=%s does not require rengeneration", keyname, locale)
         return
 
-    # finally, trigger regeneration
-    if not subp.which("locale-gen"):
-        install_function(["locales"])
+    # ensure tooling
```

Comment:
Unnecessary leading comment
```suggestion
    # ensure tooling
```